### ISSUE TITLE
Potential naming issue with import of importlib-metadata package

### ISSUE THREAD

**Issue by tilofreiwald** [timoast/sinto#53]
Hi Tim, when loading your package in python I ran into an error:

    >>> import sinto
    Traceback (most recent call last):
    File "<stdin>", line 1, in <module>
    File "/data/user/conda/envs/sinto/lib/python3.7/site-packages/sinto/__init__.py", line 1, in <module>
    import importlib.metadata
    ModuleNotFoundError: No module named 'importlib.metadata'

Then I noticed in my installation the package is called `envs/sinto/lib/python3.7/site-packages/importlib_metadata` (with an underscore instead of a dot). So changing `import.metadata` to `import_metadata` in `conda/pkgs/sinto-0.8.1-pyhfa5458b_0/site-packages/sinto/__init__.py`  resolves the error. Maybe this is a version specific naming?

Thanks!
Tilo

My conda environment:
  - _libgcc_mutex=0.1=conda_forge
  - _openmp_mutex=4.5=2_gnu
  - bzip2=1.0.8=h7f98852_4
  - c-ares=1.18.1=h7f98852_0
  - ca-certificates=2022.6.15=ha878542_0
  - curl=7.83.1=h2283fc2_0
  - keyutils=1.6.1=h166bdaf_0
  - krb5=1.19.3=h08a2579_0
  - ld_impl_linux-64=2.36.1=hea4e1c9_2
  - libblas=3.9.0=15_linux64_openblas
  - libcblas=3.9.0=15_linux64_openblas
  - libcurl=7.83.1=h2283fc2_0
  - libdeflate=1.6=h516909a_0
  - libedit=3.1.20191231=he28a2e2_2
  - libev=4.33=h516909a_1
  - libffi=3.4.2=h7f98852_5
  - libgcc-ng=12.1.0=h8d9b700_16
  - libgfortran-ng=12.1.0=h69a702a_16
  - libgfortran5=12.1.0=hdcd56e2_16
  - libgomp=12.1.0=h8d9b700_16
  - liblapack=3.9.0=15_linux64_openblas
  - libnghttp2=1.47.0=he49606f_0
  - libnsl=2.0.0=h7f98852_0
  - libopenblas=0.3.20=pthreads_h78a6416_1
  - libssh2=1.10.0=ha35d2d1_2
  - libstdcxx-ng=12.1.0=ha89aaad_16
  - libzlib=1.2.12=h166bdaf_2
  - ncurses=6.3=h27087fc_1
  - numpy=1.21.6=py37h976b520_0
  - openssl=3.0.5=h166bdaf_1
  - pip=22.2.2=pyhd8ed1ab_0
  - pysam=0.16.0=py37ha9a96c6_0
  - python=3.7.12=hf930737_100_cpython
  - python_abi=3.7=2_cp37m
  - readline=8.1.2=h0f457ee_0
  - scipy=1.7.3=py37hf2a6cf1_0
  - setuptools=63.4.1=py37h89c1867_0
  - sinto=0.8.1=pyhfa5458b_0
  - sqlite=3.39.2=h4ff8645_0
  - tk=8.6.12=h27826a3_0
  - wheel=0.37.1=pyhd8ed1ab_0
  - xz=5.2.5=h516909a_1
  - zlib=1.2.12=h166bdaf_2
  - pip:
    - importlib-metadata==4.12.0
    - typing-extensions==4.3.0
    - zipp==3.8.1

**Comment by timoast** [comment]
`importlib_metadata` is a different package, `importlib.metadata` is included in python >=3.8. The issue here is I didn’t set the minimum python version requirement, it should be 3.8. I will add that to `setup.cfg`, in the meantime installing python >=3.8 should solve this issue.

**Comment by tilofreiwald** [comment]
Oh, thanks! That worked for me.

**Comment by bellenger-l** [comment]
Hello !

I encounter the same issue when I use conda to install sinto. Even if it installs the version 0.9.0 by default, I had to specify the python version to overcome this error (and another error appears later so I also specified to install cython which seems to be required by pysam).

In order to have a functional conda environment, I had to run the following command : 

```
conda create -n sinto_env python=3.8 sinto cython
```

Thanks for this tool, I can't wait to test it.
Best
Lea
